webapp/src/dogma/features/auth/Authorized.tsx

```diff
@@ -48,7 +48,11 @@ export const Authorized = (props: { children: ReactNode }) => {
     return <>{props.children}</>;
   }
   if (typeof window !== 'undefined' && !auth.isAuthenticated) {
-    router.push(`${process.env.NEXT_PUBLIC_HOST}/link/auth/login/`);
+    if (process.env.NEXT_PUBLIC_HOST) {
+      router.push(`${process.env.NEXT_PUBLIC_HOST}/link/auth/login/?return_to=http://127.0.0.1:3000`);
```

Comment:
How about 
```
   return_to=${window.location.host}
```

Reply:
Sounds good!

Reply:
`window.location.host` does not have a protocol part.
After googling, `window.location.origin` fits our desire.
https://stackoverflow.com/a/25329802/1736581